libs/providers/multi-provider-web/README.md

```diff
@@ -0,0 +1,139 @@
+# OpenFeature Multi-Provider
+
+The Multi-Provider allows you to use multiple underlying providers as sources of flag data for the OpenFeature server SDK.
```

Comment:
```suggestion
The Multi-Provider allows you to use multiple underlying providers as sources of flag data for the OpenFeature web SDK.
```

Reply:
good catch - fixed in latest commit amendment